src/main/java/fi/aalto/cs/apluscourses/model/SubmissionResult.java

```diff
@@ -0,0 +1,70 @@
+package fi.aalto.cs.apluscourses.model;
+
+import static org.apache.commons.lang3.StringUtils.substringBetween;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class SubmissionResult {
+
+  private int exerciseId;
+  private List<Integer> submissionIds;
+  private int submissionsCount;
+  //  private int maxSubmissionsCount;
```

Comment:
waiting for [this](https://github.com/apluslms/a-plus/issues/605) to complete

Reply:
I think it's now coming to the list API instead of the points API, I can fix this in my branch